bin/p2-replicate/main.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"io/ioutil"
+	"log"
+	"os"
+
+	"github.com/armon/consul-api"
+	"github.com/square/p2/pkg/allocation"
+	"github.com/square/p2/pkg/health"
+	"github.com/square/p2/pkg/kp"
+	"github.com/square/p2/pkg/pods"
+	"github.com/square/p2/pkg/replication"
+	"github.com/square/p2/pkg/uri"
+	"github.com/square/p2/pkg/version"
+	"gopkg.in/alecthomas/kingpin.v1"
+)
+
+var (
+	replicate = kingpin.New("p2-replicate", `p2-replicate uses the replication package to schedule deployment of a pod across multiple nodes. See the replication package's README and godoc for more information.
+
+	Example invocation: p2-replicate helloworld.yaml aws{1,2,3}.example.com
```

Comment:
Suggestion: change it to
```
p2-replicate --min-nodes 2 helloworld.yaml aws{1,2,3}.example.com
```
And add an explanatory comment about exactly what the replication package would do for this command invocation.